examples/restaurant_visits/run_anonymize_values.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2022 OpenMined.
```

Comment:
Maybe we can call this file "run_preaggregated_data_example" or something similar?